### PR TITLE
[IT-2768] Update role name for CI

### PR DESCRIPTION
**Pull request by zaro0508**
The CI role names to assume will change due to PR
https://github.com/Sage-Bionetworks-IT/organizations-infra/pull/859. update to the new role names

depends on https://github.com/Sage-Bionetworks-IT/organizations-infra/pull/859

### CODE REVIEW

**Review by xschildw — APPROVED** [review]

**Review by BrunoGrandePhD — APPROVED** [review]
I'll merge once dpulls pings this PR.

**Comment by BrunoGrandePhD** [comment]
@zaro0508: Can you confirm that this PR can be merged? I saw some mention about the role name being too long in #sageit, so I wanted to make sure that this is good to go.

**Comment by zaro0508** [comment]
yeah, you have the green light @BrunoGrandePhD .

**Pull request merged**
